src/ParquetMsg.chpl

```diff
@@ -496,7 +496,7 @@ module ParquetMsg {
                  moduleName=getModuleName(), 
                  errorClass='IllegalArgumentError');
     }
-    return warnFlag;
+    return filesExist;
```

Comment:
Can we update this to match the other return statement?

Reply:
I don't think we want that in the processParquetFilenames function, only in the write dist array function.

Reply:
You are correct. I missed the return for the write and somehow thought this was it. My bad!